src/ResourceManager/DataMigration/Commands.DataMigration/Cmdlets/NewMongoDbCollectionSettingCmdlet.cs

```diff
@@ -0,0 +1,107 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Management.Automation;
+using System.Text.RegularExpressions;
+using Microsoft.Azure.Management.DataMigration.Models;
+using Microsoft.Azure.Commands.DataMigration.Models.MongoDb;
+
+namespace Microsoft.Azure.Commands.DataMigration.Cmdlets
+{
+    /// <summary>
+    /// Class that creates a new instance of the mongo db collection info
+    /// </summary>
+    [Cmdlet(VerbsCommon.New, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "DataMigrationMongoDbCollectionSetting", SupportsShouldProcess = true), OutputType(typeof(MongoDbCollectionSetting))]
+    [Alias("New-"+ ResourceManager.Common.AzureRMConstants.AzureRMPrefix +"DmsMongoDbCollectionSetting")]
+    public class NewMongoDbCollectionSettingCmdlet : DataMigrationCmdlet
+    {
+        [Parameter(
+            Mandatory = true,
+            HelpMessage = "The name of the database collection.")]
+        [ValidateNotNullOrEmpty]
+        [Alias("Name")]
+        public string CollectionName;
+
+        [Parameter(
+           Mandatory = false,
+           HelpMessage = "The database level shared RU at the target CosmosDb"
+               )]
+        [Alias("RU")]
+        public int? TargetRequestUnit { get; set; }
```

Comment:
@haitsongmsft is there a specific range of values that the user can provide? If so, I'd recommend using the `ValidateRange` attribute to limit the integers that a user could provide to this parameter.

Reply:
Good catch, the range is 400-1000000